cmd/kosli/root.go

```diff
@@ -154,6 +154,8 @@ The service principal needs to have the following permissions:
 	awsSecretKeyFlag            = "The AWS secret access key."
 	awsRegionFlag               = "The AWS region."
 	bucketNameFlag              = "The name of the S3 bucket."
+	bucketPathsFlag             = "[optional] The comma separated list of file and/or directory paths in the S3 bucket to include when fingerprinting."
```

Comment:
Should we mention that `bucketPathsFlag` takes priority over `excludeBucketPathsFlag`?

Reply:
they cannot be set together. I updated the descriptions to be more clear